styles/globals.css

```diff
@@ -181,7 +183,7 @@
   --typo-button-sm-size: 12px;
   --typo-button-sm-weight: 500;
   --typo-button-sm-line-height: 12px;
-  --typo-button-sm-letter-spacing: 0.1em;
+  --typo-button-sm-letter-spacing: 0.05em;
```

Comment:
is this taking from the design?

Reply:
In the design it's defined in pixels. But the previous implementation had the letter spacing too wide apart. So I eyeballed it to match the Figma.